auacm/app/static/js/controllers/EditCompetitionController.js

```diff
@@ -0,0 +1,223 @@
+app.controller('CreateCompetitionController', ['$scope', '$http', '$location',
+        '$routeParams', function($scope, $http, $location, $routeParams) {
+    $scope.compProblems = [];
+    var competition;
+
+    if ($routeParams.cid !== undefined) {
```

Comment:
In my code I checked for undefined variables using `typeof someVar === 'undefined'`. It seems that these two ways do [about the same thing](https://stackoverflow.com/questions/3390396/how-to-check-for-undefined-in-javascript), but I'd like to have one way that we decide and stick to. What do you think?